Lib/inspect.py

```diff
@@ -2436,15 +2446,15 @@ class Parameter:
     empty = _empty
 
     def __init__(self, name, kind, *, default=_empty, annotation=_empty):
-
-        if kind not in (_POSITIONAL_ONLY, _POSITIONAL_OR_KEYWORD,
-                        _VAR_POSITIONAL, _KEYWORD_ONLY, _VAR_KEYWORD):
+        try:
+            self._kind = _ParameterKind(kind)
+        except Exception as e:
             raise ValueError("invalid value for 'Parameter.kind' attribute")
```

Comment:
```python
except ValueError:
    raise ValueError("invalid value for 'Parameter.kind' attribute") from None
```

Reply:
But note that the second line is too long.